sirepo/template/elegant.py

```diff
@@ -1453,3 +1456,24 @@ def _sdds_init():
     _x = getattr(_s, "SDDS_LONGDOUBLE", None)
     _SDDS_DOUBLE_TYPES = [_s.SDDS_DOUBLE, _s.SDDS_FLOAT] + ([_x] if _x else [])
     _SDDS_STRING_TYPE = _s.SDDS_STRING
+
+
+def analysis_job_log_to_html(data):
+    r = simulation_db.simulation_run_dir(
```

Comment:
No need to fix here but I'll fix at some point https://github.com/radiasoft/sirepo/issues/4850